lib/cisco_node_utils/dhcp_relay_global.rb

```diff
@@ -0,0 +1,304 @@
+# September 2016, Sai Chintalapudi
+#
+# Copyright (c) 2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'node_util'
+
+module Cisco
+  # node_utils class for dhcp_relay_global
+  class DhcpRelayGlobal < NodeUtil
+    attr_reader :name
+
+    def initialize(name)
+      fail TypeError unless name.is_a?(String)
+      fail ArgumentError unless name == 'default'
+      @name = name.downcase
```

Comment:
Why do you need to downcase name here?  You generate an `ArgumentError` above if the name does not equal 'default'.